app/assets/stylesheets/base/_cams.scss

```diff
@@ -0,0 +1,22 @@
+
+.cam-model-container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 1em 0 0 0;
+}
+
+.cam-model {
+  display: inline-block;
+  flex-grow: 1;
```

Comment:
If you want to use `flex-grow` you have to have `display: flex;` in the above line